### ISSUE TITLE
 Animating single bulletpoints in OO Presentation

### ISSUE THREAD

**Issue by anfokaboom** [ONLYOFFICE/DocumentServer#2715]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

Hi
This request rides on whether I have not misunderstood helpcenter articles, but with a given text box, I can animate the text box to appear, but not the single bullet points within. Have I somehow overlooked this feature which MS PowerPoint classifies as "by paragraph"? (see attachment)

![Screenshot 2024-04-25 213617](https://github.com/ONLYOFFICE/DesktopEditors/assets/79981132/619886fc-cfaa-47ca-9b70-f307a4e843f9)

**Comment by jamesmalmeida** [comment]
Any news on this feature request?
This is basically what I need to fully migrate to ONLYOFFICE.

**Comment by Rita-Bubnova** [comment]
Sorry, currently we have no ETA.
I'm just a QA specialist, I'm not in position to gurantee any dates. You better contact sales@onlyoffice.com.

**Comment by andischmandi713** [comment]
> Any news on this feature request? This is basically what I need to fully migrate to ONLYOFFICE.

Yes, same for me, wanted to create an issue but found this. Several people I know are dependent on this feature for creating presentations for university lectures. If this is implemented they can also migrate to OnlyOffice.

**Comment by practicalwellbeing** [comment]
Same for me.

**Comment by k-soomro** [comment]
Me too please

**Comment by Botspot** [comment]
Someone on Reddit [said](https://www.reddit.com/r/OnlyOffice/comments/1jamte7/comment/nc0bpc2/) this feature is going to be released in version 9.2. Did that happen?

I'm using 9.2, and can't seem to find the feature anywhere, though if this was implemented that would be great...

**Comment by ElenaMaaya** [comment]
> Someone on Reddit [said](https://www.reddit.com/r/OnlyOffice/comments/1jamte7/comment/nc0bpc2/) this feature is going to be released in version 9.2. Did that happen?
> 
> I'm using 9.2, and can't seem to find the feature anywhere, though if this was implemented that would be great...

Hello, @Botspot !
In version 9.4 the implementation of text animation support (for lists, letters) is partial: the feature works only when opening files created in MS PowerPoint — you’ll see the animations as intended — but you currently cannot create or edit animations for list items directly within the app.
I’ll pass your request on to the development team. 
Thank you for reaching out!
